src/back/auth/src/api/token_introspect.cpp

```diff
@@ -12,14 +12,21 @@ TokenIntrospect::TokenIntrospect(
 	const components::ComponentContext& ctx)
 	: server::handlers::HttpHandlerBase{conf, ctx}
 	, _s{ctx.FindComponent<Service>()}
+	, _mapHttpMethodToSchema{LoadSchemas(kName, _s.GetJSONSchemasPath())}
 {}
 
 std::string TokenIntrospect::HandleRequestThrow(
 	const server::http::HttpRequest& req,
 	server::request::RequestContext&) const
 {
-	auto& token = req.GetCookie(Consts::SessionCookieName);
-	const std::string& userAgent = req.GetHeader(http::headers::kUserAgent);
+	const formats::json::Value body;
+	const auto params = ValidateRequest(_mapHttpMethodToSchema, req, body);
+
+	std::string token;
+	if (params.HasMember(Consts::SessionCookieName))
+		token = params[Consts::SessionCookieName].As<std::string>();
+
+	const std::string& userAgent = params[http::headers::kUserAgent].As<std::string>();
 
 	try {
 		const auto data = _s.Session().Token().Verify(token);
```

Comment:
Сейчас выглядит так как будто мы проверяем пустой токен. Кажется это делать не нужно.
Конкретно в этой ручке мы можем засунуть в общий try catch ещё и валидацию, потому что тут, как мне кажется, 400 код не актуален, от этой ручки nginx ждёт либо 2xx либо 401
```suggestion
	try {
		const auto params = ValidateRequest(_mapHttpMethodToSchema, req);
		const auto token = params[Consts::SessionCookieName].As<std::string>();		
		const auto userAgent = params[http::headers::kUserAgent].As<std::string>();
		const auto data = _s.Session().Token().Verify(token);
```

Reply:
Готово.